pages/software/modules.md

```diff
@@ -451,3 +454,34 @@ install.packages("ggplot2", repos="http://cran.r-project.org", lib = "/projects/
 If your research group requires many R libraries not already available in one of CCR's R installations, we recommend you [ask CCR to build
 them](../software/building.md#software-build-requests) or create your own custom 
 R bundle with easybuild.
+
+### SAS  
+
+SAS 9.4 is available via an Apptainer/Singularity container. You can find the SAS container here: `/util/software/containers/x86_64/sas94.sif`  This file is 4GB in size so please be sure to copy it to a location where you have enough space (i.e. your project directory), or you can run the container from this location.  
+
+!!! Warning  
+    Apptainer/Singularity is only available on compute and compile nodes.  Please refer to our [container documentation](../howto/containerization.md) for more information on using containers. 
+
+
+**SAS GUI**  
+
+To run the SAS GUI using this container, first start an [OnDemand desktop](../portals/ood.md#interactive-apps) session.  Once started, launch a terminal in the OnDemand desktop.  Then run this command to start the container and launch the SAS GUI:  
+
+```
+$ apptainer exec -B /scratch:/scratch sas94.sif /usr/local/SASHome/SASFoundation/9.4/sas
+```  
+
+!!! Tip  "Accessing additional directories in your container"  
+    The `-B` option is telling Apptainer/Singularity which directories on the node you want to bind mount into the container.  You will automatically get access to your home directory.  The directories we have listed here to bind mount are required.  If you also want access to your project directory you can add `,/projects:/projects` to the list or your specific directory `,/projects/academic/[YourGroupName]:/projects/academic/[YourGroupName]` for example.
```

Comment:
This line is repeated both here and and in the Abaqus documentation. Do we want to move it to the Containerization page and point Abaqus+SAS blurbs to the other page? It'll be easier to maintain the language that way.

Reply:
I completely agree in most circumstances.  We made a concerted effort when building this documentation to not be repetitive and be as concise as possible.  I made the choice to duplicate this here and in the container docs because I know the users of these two applications won't be interested in learning much about containers and will only want to run the software.  If others agree that it should be removed, I'll do that.